MovieApp/MovieApp/App/Features/MovieBrowser/Components/TableViewCategoryCell/CategoryCell.swift

```diff
@@ -77,3 +83,20 @@ extension CategoryCell: UICollectionViewDelegateFlowLayout {
     
 }
 
+extension CategoryCell: CategoryCellDelegate {
+    
+    func changeSubcategory(to subcategory: LocalSubcategory?) {
+        guard
+            let subcategoryMovies = subcategoryMovies,
+            let subcategory = subcategory
+        else {
+            return
+        }
+        
+        if Array(subcategoryMovies.keys).contains(subcategory) {
```

Comment:
You could've moved this to `guard` and changed it to just `let movies = subcategoryMovies[subcategory]`